### ISSUE TITLE
Smokeview Control-A does not work in fields

### ISSUE THREAD

**Issue by firemist** [firemodels/fds#10769]
When clicking in fields in smokeview, <ctrl>-<a> does not work. It is convenient to be able to hit this key sequence in order to clear out an entire field at once. This is merely a convenience item and is non-critical, but happens frequently enough to make the request.

My search did not yield any other issues on this request.

Version:
![image](https://user-images.githubusercontent.com/25644607/178787249-fda3b8d7-c111-4a5d-8c3f-66b27b711436.png)

**Comment by firemist** [comment]
I see my key sequence did not print and was apparently interpreted as html commands. Here's what it was supposed to be.

\<ctrl\>-\<a\>

And, actually, my title is a little incorrect as I meant small-a, not capital.

**Comment by firemist** [comment]
Finally, with apologies, as somehow I just can't seem to get this whole thing out in one post today, this has never worked in smokeview to my knowledge. I don't know of any version where I could just hit ctrl-a and highlight the whole field. I've been meaning to write this issue request for a long time. Thanks.

**Comment by gforney** [comment]
'CTRL a' is  not implemented in the library smokeview uses for dialog boxes.  I modified this library (wasn't easy) to use CTRL HOME instead of CTRL a for selecting an entire edit field.  for technical reasons in the way this library was implemented CTRL HOME is a lot easier to implement for this than CTRL a.  

Also note this library uses 
HOME to move the cursor to the beginning of the edit field
END to move the cursor to the end of the edit field
SHIFT while moving cursor extends the selection

CTRL HOME should be implemented in smokeview's posted here
https://drive.google.com/drive/folders/0B_wB1pJL2bFQc1F4cjJWY2duWTA?resourcekey=0-J_yHat7iJoBp1fk6QS8gMA&usp=sharing
